js/packages/cli/src/helpers/accounts.ts

```diff
@@ -142,3 +215,19 @@ export async function loadAnchorProgram(walletKeyPair: Keypair, env: string) {
 
   return new anchor.Program(idl, CANDY_MACHINE_PROGRAM_ID, provider);
 }
+
+export async function loadFairLaunchProgram(
+  walletKeyPair: Keypair,
+  env: string,
+) {
+  const solConnection = new anchor.web3.Connection(
+    `https://api.${env}.solana.com/`,
+  );
+  const walletWrapper = new anchor.Wallet(walletKeyPair);
+  const provider = new anchor.Provider(solConnection, walletWrapper, {
+    preflightCommitment: 'recent',
+  });
+  const idl = await anchor.Program.fetchIdl(FAIR_LAUNCH_PROGRAM_ID, provider);
+
+  return new anchor.Program(idl, FAIR_LAUNCH_PROGRAM_ID, provider);
```

Comment:
instead of having two very similar methods u should have 3:
- loadAnchorProgram (that has all the logic)
- loadCandyProgram(that only provides arguments to loadAnchorProgram)
- loadFairLaunchProgram(that only provides arguments to loadAnchorProgram)

Reply:
submit a PR to my PR, I'm grindin bro ;)

Reply:
right after you review my PR ;) 
https://github.com/metaplex-foundation/metaplex/pull/337
it's been out there for like 4 days